src/models/room/index.js

```diff
@@ -77,18 +77,40 @@ const Room = sequelize.define('Room', {
         return Promise.resolve(null);
       }
 
-      const [periodCoef, serviceFees] = await Promise.all([
+      const today = Utils.now();
+      const fourteenDaysFromNow = D.startOfDay(D.addDays(today, 15));
+      const nineteenDaysFromNow = D.startOfDay(D.addDays(today, 20));
+      const twentyFourDaysFromNow = D.startOfDay(D.addDays(today, 25));
+      const [periodCoef, serviceFees, city] = await Promise.all([
         models.City.getPeriodCoef({
-          date: D.max(Utils.now(), this.availableAt),
+          date: D.max(today, this.availableAt),
```

Comment:
Pour que ça match avec ce que j'ai suggéré au dessus, il faut sûrement légèrement modifier cette ligne (si l'heure a de l'importance) :
```suggestion
          date: D.max(Utils.now(), this.availableAt),
```